.github/workflows/update-cmo-jsonnet-deps.yaml

```diff
@@ -0,0 +1,58 @@
+name: Upgrade cmo jsonnet dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'
+env:
+  USER: 'github-actions[bot]<github-actions[bot]@users.noreply.github.com>'
+
+jobs:
+  jsonnet-update:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: openshift/cluster-monitoring-operator
+        ref: master
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: update jsonnet dependencies
+      run: make update
+    - name: generate yaml assets
+      run: make generate
+    - name: get pr creation app token
+      id: pr
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.APP_ID }}
+        private_key: ${{ secrets.APP_PRIVATE_KEY }}
+    - name: get cloner app token
+      id: cloner
+      uses: getsentry/action-github-app-token@v1
+      with:
+        app_id: ${{ secrets.CLONER_APP_ID }}
+        private_key: ${{ secrets.CLONER_APP_PRIVATE_KEY }}
+    - name: Render template
+      id: template
+      uses: chuhlomin/render-template@v1.2
+      with:
+        template: .github/PULL_REQUEST_TEMPLATE.md
+    - name: Create Pull Request
+      uses: arajkumar/create-pull-request@v3
```

Comment:
it may be better to keep the fork under `rhobs` itself

Reply:
It is a temporary change, I will try to contribute our changes to upstream `create-pull-request`.